content/source/docs/enterprise/admin/general.html.md

```diff
@@ -36,3 +38,17 @@ These are configurable on a global level:
 or in the Admin settings at an organization level:
 
 ![screenshot: organization run timeout page](./images/admin-org-timeout-settings.png)
+
+## Commit Statuses for Untriggered Speculative Plans
+
+This setting affects Terraform Enterprise's behavior with shared VCS repositories that contain multiple Terraform configurations.
+
+Workspaces that use part of a shared repository typically don't run plans for changes that don't affect their files; this includes [speculative plans][] on pull requests. Since "pending" status checks can block pull requests, a workspace will automatically send passing commit statuses for any PRs that don't affect its files.
+
+However, if this results in sending too many status checks to your VCS provider due to a large number of workspaces sharing one VCS repository, you can disable this behavior and ignore the pending status checks for unaffected workspaces.
+
+## Allow Speculative Plans on Pull Requests from Forks
+
+~> **Note:** This setting is currently supported for the following VCS providers: GitHub.com, GitHub.com (OAuth), GitHub Enterprise, Bitbucket Cloud, Azure DevOps Server, Azure DevOps Services.
```

Comment:
I think it would be good to add an indicator for which TFE version this setting becomes available in so people who are linked to the documentation know they need to upgrade.

Reply:
This is a good point. I've added this in the places you noted, as well as a few others.